test/test_device_radix_sort.cu

```diff
@@ -1147,11 +1313,11 @@ void TestSizes(
  */
 template <typename KeyT>
 void TestGen(
-    int             max_items,
+    size_t             max_items,
```

Comment:
We should maintain the alignment of the identifiers when changing these types.

Reply:
Done.